rules/windows/process_creation/proc_creation_win_msxsl_remote_execution.yml

```diff
@@ -0,0 +1,23 @@
+title: Remote XSL Execution Via Msxsl.EXE
+id: 75d0a94e-6252-448d-a7be-d953dff527bb
+status: experimental
+description: Detects the execution of the "msxsl" binary with an "http" keyword in the commandline. Which might indicate a potential remote execution of XSL file.
```

Comment:
```suggestion
description: Detects the execution of the "msxsl" binary with an "http" keyword in the command line. This might indicate a potential remote execution of XSL files.
```